.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!-- add an 'x' in the brackets below -->
 
-- [ ] I have added an entry to `docs/changelog.md`
+- [ ] I have added an entry to `changelog.d`
```

Comment:
```suggestion
- [ ] I have added a news fragment under `changelog.d/` (if the patch affects the end users)
```
Since many have stumbled over this before.

Reply:
This would throw off the pre-commit hook, as it checks if a news fragment has been created in the branch. In your opinion - should I get rid of the pre-commit hook, or should I create a new towncrier type for internal changes that will not be added to the changelog? I err on the side of not having the pre-commit hook, as we have this checkmark in the PR template and creating a news fragment just for it to be discarded seems unecessary.

Reply:
> should I get rid of the pre-commit hook

Yes, I think so. There can also be very minor changes for which creating changelog would be overshooting.

> I err on the side of not having the pre-commit hook, as we have this checkmark in the PR template and creating a news fragment just for it to be discarded seems unecessary.

Well, that checkmark implies, and you could also document it in the contributing guide, that a news fragment is not always needed.